src/kixi/hecuba/weather/service.clj

```diff
@@ -38,14 +58,22 @@
 (defn get-entity-ids [args-map]
   (let [entities (-> (run-api-search args-map)
                      (get-in ["entities"]))]
-    (mapv (fn [entity] (get entity "entity_id")) entities)))
+    (if (= true (:debugmode args-map))
+      (println "In debug mode, not sending to Kafka")
```

Comment:
we probably want to use real logging here so we can send things to ELK. @elisehuard should know what needs to be done here